client/my-sites/migrate/step-confirm-migration.jsx

```diff
@@ -68,20 +82,20 @@ class StepConfirmMigration extends Component {
 	}
 
 	renderMigrationButton() {
-		const { targetSite } = this.props;
+		const { targetSite, translate } = this.props;
 		const targetSiteDomain = get( targetSite, 'domain' );
 
 		if ( this.isTargetSitePlanCompatible() ) {
 			return (
 				<MigrateButton onClick={ this.handleClick } targetSiteDomain={ targetSiteDomain }>
-					Import everything
+					{ translate( 'Import everything' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **5** times:
`translate( 'Import Everything' )` *ES Score: 11*